packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -624,13 +640,23 @@ export const useNotifiSubscribe: ({
   return {
     resendEmailVerificationLink,
     instantSubscribe,
-    isAuthenticated: client.isAuthenticated,
-    isInitialized: client.isInitialized,
-    isTokenExpired: client.isTokenExpired,
+    isAuthenticated: client?.isAuthenticated ?? false,
+    isInitialized: client?.isInitialized ?? false,
+    isTokenExpired: client?.isTokenExpired ?? false,
     logIn,
     subscribe,
     updateTargetGroups,
     subscribeWallet,
     updateWallets,
   };
 };
+
+const dummySubscribeData: SubscriptionData = {
+  alerts: { dummy: {} as unknown as Alert },
+  email: null,
+  phoneNumber: null,
+  telegramId: null,
+  telegramConfirmationUrl: null,
+  isPhoneNumberConfirmed: null,
+  emailIdThatNeedsConfirmation: '',
+};
```

Comment:
in general, having "dummy" data spread across many files is an architectural smell, ideally we can have it in one place (like a DummySubscriptionContext or something) and have others cleanly consuming

maybe it's time to re-think the dependencies between useNotifiSubscribe, useNotifiClient, etc. etc....

Reply:
We no longer need the `dummySubscribeData` because we throw error instead of returning dummy value. 
And Already Migrated demoPreview to newly created 'NotifiDemoPreviewContext' that includes related demoPreview consts and types.